confgenerator/nginx.go

```diff
@@ -99,6 +101,25 @@ func (p LoggingProcessorNginxAccess) Components(tag string, uid string) []fluent
 			},
 		})
 	}
+	// Assign severity based on HTTP status code class (enables use of system
```

Comment:
This is very, very weird to me. "5xx" responses are arguably errors, but the other levels are all normal responses.